src/views/TimezoneModal.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <ion-header>
+     <ion-toolbar>
+        <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+     <ion-title>{{ $t("Select time zone") }}</ion-title>
+    </ion-toolbar>
+    <ion-toolbar>
+        <ion-searchbar @ionFocus="selectSearchBarText($event)" :placeholder="$t('Search time zones')"  v-model="queryString" @ionInput="findTimeZone()" />
+    </ion-toolbar>
+  </ion-header>
+  <ion-content class="ion-padding">
+    <!-- Empty state -->
+    <div class="empty-state" v-if="filteredTimeZones.length === 0">
+      <p>{{ $t("No time zone found")}}</p>
+    </div>
+
+    <!-- Timezones -->
+    <div v-else>
+      <ion-list>
+        <ion-radio-group value="rd" v-model="timeZoneId">
+          <ion-item  v-bind:key="timeZone.id" v-for="timeZone in filteredTimeZones">
```

Comment:
When using `:` we don't need to use `v-bind`, `:` is shorthand for `v-bind`.

Reply:
Done Sir